pkg/kubelet/apis/podresources/server_v1.go

```diff
@@ -27,22 +27,24 @@ import (
 	"k8s.io/kubelet/pkg/apis/podresources/v1"
 )
 
-// podResourcesServerV1alpha1 implements PodResourcesListerServer
+// podResourcesServerV1 implements PodResourcesListerServer
 type v1PodResourcesServer struct {
-	podsProvider    PodsProvider
-	devicesProvider DevicesProvider
-	cpusProvider    CPUsProvider
-	memoryProvider  MemoryProvider
+	podsProvider             PodsProvider
+	devicesProvider          DevicesProvider
+	cpusProvider             CPUsProvider
+	memoryProvider           MemoryProvider
+	dynamicResourcesProvider DynamicResourcesProvider
```

Comment:
It's still unclear to me why we can't just include  a single `PodResourcesProviders` field here as previously suggested. Please comment.

Reply:
Ack